File: DungeonDebt/Assets/Scripts/Data/PayrollActionDefinition.cs
public class PayrollActionDefinition
{
    public PayrollActionDefinition(
        PayrollActionId id,
        string displayName,
        string description)
    {
        Id = id;
        DisplayName = displayName;
        Description = description;
    }

    public PayrollActionId Id { get; }
    public string DisplayName { get; }
    public string Description { get; }
}

File: web/src/data/PayrollActionDefinition.js
// Ported from DungeonDebt/Assets/Scripts/Data/PayrollActionDefinition.cs
export class PayrollActionDefinition {
  constructor(id, displayName, description) {
    this.id = id;
    this.displayName = displayName;
    this.description = description;
    Object.freeze(this);
  }
}
